modules/experimental-layers/src/tile-layer/tile-layer.js

```diff
@@ -43,9 +45,12 @@ export default class TileLayer extends CompositeLayer {
           const allCurrTilesLoaded = currTiles.every(tile => tile.isLoaded);
```

Comment:
Currently in Tile class, if loading errors out, then `isLoaded` will never be set. This causes unnecessary overhead every time the viewport changes. I wonder if it makes more sense to make `isLoaded` true and keep `_data` empty in this case.